operation.go

```diff
@@ -0,0 +1,6 @@
+package merkletree
+
+type Operation struct {
```

Comment:
I'm thinking that ```Operation``` should maybe be moved outside of the merkle prefix tree entirely. Right now, it's not doing much since it's only a wrapper for a key-value pair, and all operations (registrations and changes) are being treated as inserts (which is fine). Since the CONIKS server will be handling registration and key change operations, and it will have to order these, it may make more sense to define ```Operation``` at the server level only?